src/terrain/clients/search.clj

```diff
@@ -1,17 +1,19 @@
 (ns terrain.clients.search
-  (:use [terrain.util.config :only [search-base-url]])
+  (:use [terrain.util.config :only [search-base-url]]
+        [terrain.util.transformers :only [secured-params]])
   (:require [clj-http.client :as http]
-            [cemerick.url :refer [url]]
-            [cheshire.core :as json]))
+            [cemerick.url :refer [url]]))
 
 (defn do-data-search
-  [params body]
-  (let [req-options  {:body         (json/encode body)
-                      :query-params params
-                      :as           :stream
-                      :content-type "application/json"}]
-    (http/post (str (url (search-base-url) "data" "search")) req-options)))
+  [body]
+  (let [req-options  {:form-params  body
```

Comment:
I think `http/post` will automatically format the `:form-params` as a JSON body because of this combination:
```clojure
:as           :json
:content-type :json
```
@slr71 explained it better in a previous PR, but I can't remember which one now 😖

Reply:
Oh, yeah. As long as the content type is `:json`, `clj-http` will serialize the contents of the `:form-params` key, and put it in the request body. I usually use that option because it means that I don't have to do the serialization explicitly which is convenient. The [clj-http docs](https://github.com/dakrone/clj-http#post) have some more information about this.